Keas.Mvc/ClientApp/components/Equipment/EquipmentEditValues.tsx

```diff
@@ -148,6 +148,23 @@ export default class EquipmentEditValues extends React.Component<IProps, {}> {
                                 onChange={e => this.props.changeProperty("model", e.target.value)}
                             />
                         </div>
+                    )}
+                    {this._shouldShowForType(this.props.selectedEquipment.type, "BigfixId") && (
+                        <div className="form-group">
+                            <label>Bigfix Id</label>
+                            <input
+                                type="text"
+                                className="form-control"
+                                disabled={this.props.disableEditing}
+                                value={
+                                    this.props.selectedEquipment.bigfixId
+                                        ? this.props.selectedEquipment.bigfixId
+                                        : ""
```

Comment:
Just as an FYI you can simplify this to `this.props.selectedEquipment.bigfixId || ""` if you want